app/models/decorators/mvi_user_decorator.rb

```diff
@@ -22,10 +22,11 @@ def create
       mvi: response
     }
     @user
-  rescue MVI::RecordNotFound => e
+  rescue MVI::RecordNotFound
     # TODO(AJD): add metric
     Rails.logger.error "MVI record not found for user: #{@user.uuid}"
-    raise Common::Exceptions::RecordNotFound, "User not in found MVI: #{e.message}"
+    @user.attributes = {}
+    # raise Common::Exceptions::RecordNotFound, "User not in found MVI: #{e.message}"
```

Comment:
This makes sense to me, but it really seems like MVI should not be treated as a dependency in the first place, and that any exception raising should happen in the authorization steps, based on the the availability of the MVI record (if necessary).

Finally, Common::Exceptions::RecordNotFound and Common::Exceptions::ValidationErrors are ideally meant for something we can inform the user to take further action upon. Since they have no way taking action on their MVI credentials, an error like ServiceOutage, or Authorization failed, or an InternalServer 500 might be more relevant -- but again, I think the authorization step should handle that on client to client basis, as not all clients require MVI creds.